Mathlib/Data/Real/GoldenRatio.lean

```diff
@@ -90,6 +90,10 @@ theorem gold_sub_goldConj : φ - ψ = Real.sqrt 5 := by
   ring
 #align gold_sub_gold_conj gold_sub_goldConj
 
+theorem gold_exp (n : ℕ) : φ ^ (n + 2) - φ ^ (n + 1) = φ ^ n := by
```

Comment:
This should probably be called something like `gold_pow_sub_gold_pow` or `gold_pow_recurrence` (the former is more in line with Mathlib naming conventions). But `exp` references things like [Complex.exp](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Data/Complex/Exponential.html#Complex.exp), or [Real.exp](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Data/Complex/Exponential.html#Real.exp) or [NormedSpace.exp](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Analysis/NormedSpace/Exponential.html#NormedSpace.exp).

You can refer to the [naming conventions](https://leanprover-community.github.io/contribute/naming.html) for more details.